docs/site/Self-hosted-REST-API-Explorer.md

```diff
@@ -0,0 +1,13 @@
+---
+lang: en
+title: 'Self-hosted REST API Explorer'
+keywords: LoopBack 4.0, LoopBack 4
+layout: readme
+source: loopback-next
+file: packages/rest-explorer/README.md
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Self-hosted-rest-api-explorer.html
+summary:
+  A tutorial on using `@loopback/rest-explorer` to add a self-hosted REST API
+  Explorer
+---
```

Comment:
Are you planning to add docs to this page in this PR?

Reply:
This page will automatically include the contents of `packages/rest-explorer/README.md`, see https://loopback.io/doc/en/lb4/Creating-decorators.html as an example of a page based on README contents.